tests/roles/backend_services/templates/openstack_version.j2

```diff
@@ -4,6 +4,7 @@ metadata:
   name: openstack
   namespace: openstack
 spec:
+  targetVersion: 0.0.1
```

Comment:
testing with hard-coded version for now. if it works i can add a task to fetch and apply with jinja variable